packages/docs-ui/src/components/SideNavigation/SideNavigationSection.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, HiddenVisually, Stack, Text } from 'braid-design-system';
+import * as styles from './SideNavigation.css';
+import React from 'react';
+import { SideNavigationItem } from './SideNavigationItem';
+
+interface SideNavigationSection {
+  title: string;
+  hideTitle?: boolean;
+  items: SideNavigationItem[];
+}
+
+export const SideNavigationSection = ({
+  title,
+  hideTitle,
+  items,
+}: SideNavigationSection) => (
+  <Box component="nav">
+    <Stack space="small">
+      {hideTitle ? (
+        <HiddenVisually>
+          <Text size="xsmall" weight="medium" component="h2">
+            {title}
+          </Text>
+        </HiddenVisually>
+      ) : (
+        <Box className={styles.uppercase}>
+          <Text size="xsmall" weight="medium" component="h2">
+            {title}
+          </Text>
+        </Box>
+      )}
```

Comment:
This will still have the undesired outcome of visible white space when the title is hidden. Moving the hidden title out of the Stack is one way to fix this.

```suggestion
    {hideTitle ? (
      <HiddenVisually>
        <Text size="xsmall" weight="medium" component="h2">
          {title}
        </Text>
      </HiddenVisually>
    ) : (
      null
    )}
    <Stack space="small">
      {!hideTitle ? (
        <Box className={styles.uppercase}>
          <Text size="xsmall" weight="medium" component="h2">
            {title}
          </Text>
        </Box>
      ) : (
        null
      )}
```

Reply:
I've pushed a new version to check. Not 100% sure if I've done the types correctly